packages/cheqd/src/dids/CheqdDidRegistrar.ts

```diff
@@ -25,8 +25,14 @@ import {
   getKeyFromVerificationMethod,
   JsonTransformer,
   VerificationMethod,
+  SECURITY_JWS_CONTEXT_URL,
 } from '@credo-ts/core'
 
+import {
+  ED25519_SUITE_CONTEXT_URL_2018,
+  ED25519_SUITE_CONTEXT_URL_2020,
+} from '../../../core/src/modules/vc/data-integrity/signature-suites/ed25519/constants'
```

Comment:
this won't work on publish, as it's importing from another package

Reply:
Thanks Timo.. this is updated and I have merged latest 0.5.x branch as well

Reply:
@TimoGlastra Should we create a separate PR targeting main branch too?

Reply:
Yes that'd be great @DaevMithran